content/files/app-activation-sequence-diagram.txt

```diff
@@ -0,0 +1,41 @@
+materialdesignicons f004 "<color:#11324d>**++User++**</color>" as User #11324d
+participant "<color:#9452ba>**++PIM++**</color>" as PIM
+participant "<color:#4f5bc6>**++App++**</color>" as App
+
+
+lifelinestyle User #11324d:1:dashed
+lifelinestyle PIM #9452ba:2:solid
+lifelinestyle App #4f5bc6:1:solid
+
+activecolor PIM #9452ba
+activecolor App #4f5bc6
+
+autonumber 1
+
+rbox over User: Initiate Activation
+User->PIM:Clicks on "connect"
+
+PIM-#9452ba:2>App: Redirection to the activation URL
+activate App
+rbox over App: App prerequisites:\ninformation, pricing, signup
+App-#4f5bc6:2>PIM:Authorization Request
+deactivate App
+activate PIM
+rbox over PIM: User consent\nPermisions configuration
+
+User->PIM: Confirm Authorization
+destroysilent User
+
+PIM-#9452ba:2>App: Authorization Response
+deactivate PIM
+
+entryspacing 2
+App-#4f5bc6>>PIM: Access Token Request
+entryspacing 0.5
+PIM--#9452ba>>App: Access Token Response
+entryspacing 2
+
+rbox over PIM, App: App is activated and ready
+
+App--#4f5bc6>>PIM: API calls with token
+activate PIM
```

Comment:
@RiffVentura 
 
 I'm copy/pasting comments I've already done on slack. We can discuss everything.
 
 - I would remove the `Initiate Activation`, it duplicate the action described in `Clicks on "connect"`
 - can you remove the autonumber ?
 - I'm not a big fan of the colors, it contrast a lot with everything else in the documentation
 - We should not talk about `Permissions configuration`, it doesn't concern the App Developer
 - `User consent` and `Confirm Authorization` are kinda duplicates
 - I would remove the `App prerequisites` block. They don't have to do it here, they can also do those steps after receiving the token. Also, those steps are on a case-by-case basis, I don't think we should talk about it, at least in the first version.
 - There should be a bold rectangle between `Access Token Request` and `Access Token Response` to show where time is spent during the full process.
 - I would remove the block `App is activated and ready`. For Apps Developers, it doesn't make sense. This is Julia's point of view where an App is activated.
 - `API calls with token`, first we shoud say `REST API`, not `API`. Then, I kinda like how Google worded this step: `Use token to call Google API`, in our case, we could do `Use token to call REST API`

Reply:
Also, Willy proposed to replace `Confirm Authorization` by `Confirm scope authorization`.
I don't think we should talk about scopes here, but yes, we should improve the wording of this step.